#include <exception>
#include <string>

using SocketException = std::runtime_error;